docs/source/user-guide/guide_api.rst

```diff
@@ -206,3 +257,80 @@ description or notes column.
          ]
        }
 
+How to return a filtered list of datasets using the DKAN search endpoint.
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

Comment:
the ^^^ needs to match the length of the heading

Reply:
Based on the feedback, I adjusted the headings so this one doesn't exist as it did -